src/OpenSage.Game/Graphics/Rendering/RenderPipeline.cs

```diff
@@ -224,8 +224,20 @@ public void Execute(RenderContext context)
 
             var standardPassCameraFrustum = scene.Camera.BoundingFrustum;
 
+            commandList.PushDebugGroup("Terrain");
+            RenderedObjectsOpaque += DoRenderPass(context, commandList, _renderList.Terrain, standardPassCameraFrustum, cloudResourceSet);
+            commandList.PopDebugGroup();
+
+            commandList.PushDebugGroup("Road");
+            RenderedObjectsOpaque += DoRenderPass(context, commandList, _renderList.Road, standardPassCameraFrustum, cloudResourceSet);
+            commandList.PopDebugGroup();
+
+            commandList.PushDebugGroup("Water");
+            DoRenderPass(context, commandList, _renderList.Water, standardPassCameraFrustum, cloudResourceSet);
```

Comment:
Why no `RenderedObjectsOpaque +=` here?

Reply:
I'm not sure, I copied this from #353...maybe water doesn't count as opaque?